pkg/proxy/proxy_test.go

```diff
@@ -9,22 +9,25 @@ import (
 	"log"
 	"net/http"
 	"net/http/httptest"
-	"net/url"
 	"testing"
 	"time"
 
-	"github.com/codeready-toolchain/registration-service/pkg/application/service"
+	service2 "github.com/codeready-toolchain/registration-service/pkg/application/service"
```

Comment:
`appservice` and `proxyservice` would be better aliases, but it's not a blocker

Reply:
I'll update it in following PR - to not lose the 20 minutes of running the e2e tests :innocent:

Reply:
anyway, I figured that I missed a header in two test cases, and since the review meeting already started then I updated the alias right away:  1149fa4